storage/aptosdb/src/transaction_store/mod.rs

```diff
@@ -316,6 +317,65 @@ impl TransactionStore {
         Ok(())
     }
 
+    /// Prune the transaction accumulator between a range of version in [begin, end).
+    ///
+    /// To avoid always pruning a full left subtree, we uses the following algorithm.
+    /// For each leaf with an odd leaf index.
+    /// 1. From the bottom upwards, find the first ancestor that's a left child of its parent.
+    /// (the position of which can be got by popping "1"s from the right of the leaf address).
+    /// Note that this node DOES NOT become non-useful.
+    /// 2. From the node found from the previous step, delete both its children non-useful, and go
+    /// to the right child to repeat the process until we reach a leaf node.
+    /// More details are in this issue https://github.com/aptos-labs/aptos-core/issues/1288.
+    pub fn prune_transaction_accumulator(
```

Comment:
I think we can also remove the get_min_proof_node function below which was used for the old algorithm?

Reply:
Done